intel_extension_for_transformers/llm/library/jblas/.clang-format

```diff
@@ -0,0 +1,7 @@
+Language:        Cpp
```

Comment:
is this file necessary?

Reply:
multiple .clang-format files are not a problem.